src/wagtail_personalisation/locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,320 @@
+msgid "Wednesday"
+msgstr ""
+
+#: rules.py:109
+msgid "Thursday"
+msgstr ""
+
+#: rules.py:110
+msgid "Friday"
+msgstr ""
+
+#: rules.py:111
+msgid "Saturday"
+msgstr ""
+
+#: rules.py:112
+msgid "Sunday"
+msgstr ""
+
+#: rules.py:136
+msgid "Day Rule"
+msgstr ""
+
+#: rules.py:152
+msgid "These users visit on"
+msgstr ""
+
+#: rules.py:153 rules.py:192 rules.py:400
+msgid "{}"
+msgstr ""
+
+#: rules.py:168
+msgid "Regex string to match the referer with"
+msgstr ""
+
+#: rules.py:187
+msgid "Referral Rule"
+msgstr ""
+
+#: rules.py:191
+msgid "These visits originate from"
+msgstr ""
+
+#: rules.py:211
+msgid "More than"
+msgstr ""
+
+#: rules.py:212
+msgid "Less than"
+msgstr ""
+
+#: rules.py:213
+msgid "Equal to"
+msgstr ""
+
+#: rules.py:270
+msgid "Visit count Rule"
+msgstr ""
+
+#: rules.py:274
+msgid "These users visited {}"
+msgstr ""
+
+#: rules.py:277
+msgid "{} {} times"
+msgstr ""
+
+#: rules.py:294
+msgid "The query parameter to search for"
+msgstr ""
+
+#: rules.py:296
+msgid "The value of the parameter to match"
+msgstr ""
+
+#: rules.py:317
+msgid "Query Rule"
+msgstr ""
+
+#: rules.py:321
+msgid "These users used a url with the query"
+msgstr ""
+
+#: rules.py:322
+msgid "?{}={}"
+msgstr ""
+
+#: rules.py:340
+msgid "Mobile phone"
+msgstr ""
+
+#: rules.py:341
+msgid "Tablet"
+msgstr ""
+
+#: rules.py:342
+msgid "Desktop"
+msgstr ""
+
+#: rules.py:367
+msgid "Device Rule"
+msgstr ""
+
+#: rules.py:391
+msgid "Logged In Rule"
+msgstr ""
+
+#: rules.py:394
+msgid "Logged in"
+msgstr ""
+
+#: rules.py:396
+msgid "Not logged in"
+msgstr ""
+
+#: rules.py:399
+msgid "These visitors are"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:12
+msgid "Filter"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:28
+msgid "This segmented has been visited"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:29
+msgid "time"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:32
+msgid "This segment has been active for"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:33
+msgid "day"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:41
+msgid "The visitor must match"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:43
+msgid "Any rule"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:45
+msgid "All rules"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:50
+msgid "The persistence of this segment is"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:52
+msgid "This segment persists in between visits"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:52
+msgid "Persistent"
+msgstr ""
+
+#: templates/modeladmin/wagtail_personalisation/segment/index.html:54
+msgid "This segment is reevaluated on every visit"
```

Comment:
This segment is reevaluated on every visit -> This segment is re-evaluated on every visit